### ISSUE TITLE
add common fields for Custom Records

### ISSUE THREAD

**Issue by ShawnTalbert** [ExploreConsulting/netsuite-fasttrack-toolkit-ss2#31]
Either generate fields such as `isinactive` and `name` for all Custom Records (codegen) or provide a base class that declares them.

**Comment by steveklett** [comment]
I plan to work on this, is anyone else already working on it?

**Comment by ShawnTalbert** [comment]
Not that I know of.

**Comment by steveklett** [comment]
@ShawnTalbert - tagging you because I'm unable to do a pull request for this new work.

This is done and committed, Extended CustomRecord.xsl to:

1.  support common fields
2. Insert an underscore for the class name if the record name starts with a number (e.g., Record named "#PL Config" would generate class name "_3PLConfig", however the filename will remain unchanged
3. Added Record name and description to class definition header comment
4. Removed a couple superfluous bits and pieces

Note: I may have "git'ed" incorretly: I had an existing pull request on my "geo2" branch. I then did this NFT NSDAL work on the same branch (oops!) and now I can't create a new pull request. It seems pull requests are branch-based rather than commit based. I think if you accept my other pull request you'll get this NFT NSDAL work as well.
